JakinChan200/CS180_Roald
# Add react-devtools

## Overview
Adds `react-devtools` to the client so that we can measure performance with a profiler across devices.
If you use Chrome or Firefox, you can find React Devtools as a browser extension. Safari needs the standalone react-devtools command.


## Materials
- N/A


## Pictures
<img width="796" alt="Screen Shot 2022-05-06 at 11 51 11 AM" src="https://user-images.githubusercontent.com/43893085/167200864-347d2929-ee7a-4ec6-b0ba-17faae2dbb6c.png">


## Manual Testing Guide
- Navigate to the `client` directory
- Install new packages with `npm install`
- Run the devtools `react-devtools`
- An electron app should open, now go to the `Profiler` tab
- Click the recording button and go to the page, complete actions of interest, then stop recording
- Click on each component in the tree to see time per render

Pull requests should receive approval before merge.
